xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -76,6 +76,33 @@ void CPVRRecordings::Update()
   CServiceBroker::GetPVRManager().PublishEvent(PVREvent::RecordingsInvalidated);
 }
 
+void CPVRRecordings::UpdateInProgressSize()
+{
+  CSingleLock lock(m_critSection);
+  if (m_bIsUpdating)
+    return;
+  m_bIsUpdating = true;
+  lock.Leave();
+
+  CLog::LogFC(LOGDEBUG, LOGPVR, "Updating recordings size");
+  bool haveUpdatedInProgessRecording = false;
+  for (auto& recording : m_recordings)
```

Comment:
const auto&

Reply:
Why const auto if the size will be updated?

Reply:
Ooops. nvm.